testdata/config/azure_key_bc.yaml

```diff
@@ -17,14 +17,6 @@ libfuse:
 
 block_cache:
   block-size-mb: 8
-  mem-size-mb: 4192
-  
-  path: block_cache
-  disk-size-mb: 4192  
-  disk-timeout-sec: 30
-
-  prefetch: 120
-  parallelism: 128
```

Comment:
I think we should only remove the disk caching part here and keep mem-size-mb, prefetch and parallelism?

Reply:
We want all of them to be autotuned hence its removed I guess.